inc/js_scripts/admin/main.js

```diff
@@ -43,8 +72,12 @@
 				alert( "Settings saved" );
 			} );
 		}
-	} )
+	} );
 
-	let view = new SettingsViewClass( {model: model, el: '.post_modifier-settings'} )
+	let model = new Backbone.Model( post_modifier.settings );
+	console.log( $( '.post-content-settings' ) );
+	let contentView = new PostContentView( {model: model, el: '.post-content-settings'} );
+	let metadataView = new PostMetadataView( {model: model, el: '.post-metadata-settings'} );
+	let main = new MainView( {model: model, el: '.post_modifier-settings', attributes: {content: contentView, metadata: metadataView,}} );
```

Comment:
You can also declare variables like this, I think it's a bit more readable ( the way you did it is ok too )
```suggestion
	let contentView = new PostContentView( {model: model, el: '.post-content-settings'} ),
	 metadataView = new PostMetadataView( {model: model, el: '.post-metadata-settings'} ),
	 main = new MainView( {model: model, el: '.post_modifier-settings', attributes: {content: contentView, metadata: metadataView,}} );
```